src/main/java/jimmy/mcgymmy/logic/commands/UndoCommand.java

```diff
@@ -2,6 +2,9 @@
 
 import jimmy.mcgymmy.model.Model;
 
+/**
+ * Undo the previous action cause by other commands (except help).
```

Comment:
Maybe it is better to be Undo the previous action that caused changes to ModelManager?

Reply:
Ok I've changed it already